Templates/QueueTrigger-Python/function.json

```diff
@@ -7,7 +7,7 @@
       },
       {
         "type": "table",
```

Comment:
So what is everyone else using, 4 spaces? Both my machines are using 2 (VS Enterprise) and I don't recall changing any defaults.